docs/en/operations/settings/settings_users.md

```diff
@@ -54,27 +54,27 @@ Password could be specified in plaintext or in SHA256 (hex format).
 
 ### user_name/networks
 
-List of networks which the user can connect to ClickHouse server from.
+List of networks that the user can connect to the ClickHouse server from.
```

Comment:
I would replace to `List of networks from which the user can connect to the Clickhouse server.`